From 1f24244080848d04aaaf86a900bb56c538b98cfb Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Thu, 16 Mar 2023 19:33:03 +0530
Subject: [PATCH 106/281] OpenWrt: 261-lib-arc4-unhide.patch

lib/crypto: add tristate string for ARC4

This makes it possible to select CONFIG_CRYPTO_LIB_ARC4 directly. We
need this to be able to compile this into the kernel and make use of it
from backports.

Signed-off-by: Felix Fietkau <nbd@nbd.name>

Change-Id: Ibac6150238888a48a2b6d793b704b6bb9efb50c3
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
---
 lib/crypto/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/crypto/Kconfig b/lib/crypto/Kconfig
index 7e9683e9f5c6..0ba9394a9469 100644
--- a/lib/crypto/Kconfig
+++ b/lib/crypto/Kconfig
@@ -9,7 +9,7 @@ config CRYPTO_LIB_AES
 	tristate
 
 config CRYPTO_LIB_ARC4
-	tristate
+	tristate "ARC4 cipher library"
 
 config CRYPTO_ARCH_HAVE_LIB_BLAKE2S
 	bool
-- 
2.17.1

